src/components/Reports/sharedComponents/ReportHeader/ReportHeader.jsx

```diff
@@ -2,17 +2,18 @@ import classnames from 'classnames';
 import { ReportBlock } from '../ReportBlock';
 import './ReportHeader.css';
 
-export const ReportHeader = ({ children, isActive, src}) => {
+export const ReportHeader = ({ children, isActive, src, avatar }) => {
   return (
     <ReportBlock>
       <header className='report-header'>
         <div className='report-header-details'>
           <div className="report-header-profile-pic-wrapper">
-            <img src={src || '/pfp-default.png'} alt="Report profile picture" className="report-header-profile-pic" />
-            <div className={classnames('report-header-activity', {active: isActive})} />
+            {avatar ? avatar : <img src={src || '/pfp-default.png'} alt="Report profile picture" className="report-header-profile-pic" />}
```

Comment:
```suggestion
            {avatar ?? <img src={src || '/pfp-default.png'} alt="Report profile picture" className="report-header-profile-pic" />}
```

Nit-pick: This reduces the need for a ternary and gives you a backup

Reply:
Yep, thank you! I updated the code